src/hb-ot-stat-table.hh

```diff
@@ -223,10 +204,44 @@ struct AxisValue
   DEFINE_SIZE_UNION (2, format);
 };
 
+struct StatAxisRecord
+{
+  inline bool has_data () const { return axisTag; }
+
+  inline int cmp (hb_tag_t a) const { return axisTag.cmp (a); }
+
+  inline unsigned int get_ordering () const { return axisOrdering; }
+
+  inline bool sanitize (hb_sanitize_context_t *c) const
+  {
+    TRACE_SANITIZE (this);
+    return_trace (likely (c->check_struct (this)));
+  }
+
+  protected:
+  Tag		axisTag;	/* A tag identifying the axis of design variation. */
+  NameID	axisNameID;	/* The name ID for entries in the 'name' table that
+				 * provide a display string for this axis. */
+  HBUINT16	axisOrdering;	/* A value that applications can use to determine
+				 * primary sorting of face names, or for ordering
+				 * of descriptors when composing family or face names. */
```

Comment:
Typically, members of struct `StatAxisRecord` don't need to have `axis` prefix.  Makes more readable.

Reply:
Thanks. Done